react-pallet-uniques/readme.md

```diff
@@ -13,6 +13,7 @@ This app is built using following tools:
 REACT_APP_NODE_WS_PROVIDER=ws://localhost:9944
 REACT_APP_FARMER_WS_PROVIDER=ws://localhost:9955
 ```
+- `npm install` or `yarn` to install dependencies
```

Comment:
Can we use just NPM and not both? And while we're at it use `npm ci`, so examples continue to work (semantic versioning is not well respected in NPM ecosystem in my experience and things can suddenly break).

Reply:
since it is based on create-react-app, I am following their instructions - https://create-react-app.dev/docs/getting-started

Reply:
I understand, but if we want examples to work tomorrow we really need to rely on lock files, which are different for NPM and Yarn.